spec/db/models.rb

```diff
@@ -22,6 +22,7 @@ class Author < VirtualTotalTestBase
   # virtual total using through
   virtual_total :total_bookmarks, :bookmarks
   alias v_total_bookmarks total_bookmarks
+  virtual_total :total_co_books, :co_books
```

Comment:
Might be useful to have a comment like the other virtual totals above

```suggestion
  # virtual total using has and belongs to many
  virtual_total :total_co_books, :co_books
```